ci/run_playbooks.yml

```diff
@@ -8,15 +8,50 @@
   vars:
     fvt_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}"
     ci_framework_logs: "{{ ansible_env.HOME }}/ci-framework-data/logs"
+    patch_openstackversions: true
   tasks:
     - name: "Create log dir"
       ansible.builtin.file:
         path: "{{ ci_framework_logs }}/functional-tests"
         state: directory
         mode: "0755"
 
+       # TODO: Convert this to a kustomization and pass it to the deployment
+       # First step might be to use -k in the patch to pass a kustomization to oc patch
+    - name: Patch the openstackversions to use the master containers for aodh and heat
```

Comment:
Are we plan to add this to vars_functional_test.yml with  cifmw_edpm_prepare_kustomizations ?

Reply:
Initially, that was the plan, since it would speed up the test. However, it would take away some flexibility to combine logging and autoscaling tests.

Perhaps we look at using the method that Jaromir used in https://github.com/openstack-k8s-operators/telemetry-operator/pull/389/files#diff-c25a30bd1ea58eb09953e3fcc124fed394ecee980c3fc0b734cc2b9bf5cf0fe4, which uses a file with a kustomization.

The decision is still pending.